tomls/omnibus-base-goerli-andromeda/perps/eth.toml

```diff
@@ -17,7 +17,7 @@ defaultValue = "0.0002"
 defaultValue = "0.0005"
 
 [setting.perpsEthMaxMarketSize]
-defaultValue = "525" # approx $1m
+defaultValue = "5250"
 
```

Comment:
We also need to increase collateral limit for usdc to back the perps market

Reply:
sure will look into it